src/it/projects/profile-with-properties-active/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.codehaus.mojo.flatten.its</groupId>
+  <artifactId>profile-with-properties-active</artifactId>
+  <version>0.0.1${versionAddon}-SNAPSHOT</version>
+
```

Comment:
This will produce warnings in Maven so I have my doubts to support such wrong setups.. Using Maven 3.5.0 officially supports a usage of `${revision}`, `${changelist}` and `${sha1}`.